terraform/resources/api.tf

```diff
@@ -71,6 +71,63 @@ resource "azurerm_linux_web_app" "nbs_appts_app" {
   }
 }
 
+resource "azurerm_monitor_autoscale_setting" "nbs_appts_sp_autoscale" {
```

Comment:
Suggest we use "scaling" as the short identifier for scaling resources - so the resource would become nbs_appts_scaling and the name (line 76) would be ${var.application_short}-scaling-${var.environment}-${var.loc}. This would make it more consistent with the naming we've used for other resources (unless their are naming rules for these resources that prevent this)

Reply:
changed accepted and code updated